translations/messages.fr.yaml

```diff
@@ -188,6 +188,20 @@ user:
   availabilityCurrentWeek: Mes disponibilités pour la semaine courante
   availabilityNextWeek: Mes disponibilités pour la semaine prochaine
   availabilitySpecificWeek: Mes disponibilités pour la semaine du %week%
+  calendar: 
+    currentWeek: Semaine courante
+    defaultAvailabilityTitle: Dispo
+    editAvailabilities: Modifier mes disponibilités
+    edit: Modifier
+    nextWeek: Semaine prochaine
+    shortDays:
+      Mon: lun
```

Comment:
why not using `| date_format('short')` or something like that ?

Reply:
I didn't find something that's ready without adding next deps to the project, as its not the core of this PR IMO we should consider that as an separated improvement. 
PS: short is providing the full date in short syntax and not only the day

Reply:
You can use `| format_date(pattern="eee")`